receiver/README.md

```diff
@@ -68,9 +68,6 @@ The full list of settings exposed for this receiver are documented [here](https:
 with detailed sample configurations [here](https://github.com/open-telemetry/opentelemetry-service/blob/master/receiver/opencensusreceiver/testdata/config.yaml).
 
 ### Writing with HTTP/JSON 
-// TODO(ccaraman) The cors setting wasn't ported accidentally. In a follow up
```

Comment:
The next line is a bit outdated, like referring to OpenCensus. Could you reword it to mention Otel and make it clearer that the receiver handles both HTTP and gRPC.

Reply:
A bit unclear here - is calling it the OpenCensus receiver incorrect? My understanding was that it's still being referred to as the OpenCensus receiver inside of Otel

Reply:
🤦‍♀ apologies my coffee is fading and rereading it a few more times I see that I was thinking about it in the wrong context. 
Ignore this comment!